src/app/routes/cpsAsset/utils/getAdditionalPageData.js

```diff
@@ -17,6 +20,61 @@ import fetchPageData from '../../utils/fetchPageData';
 
 const noop = () => {};
 
+// 004_brasil_recommendations_experiment
+const getRecommendations = (service, variant, assetUri) => {
```

Comment:
I think this logic should be split into a seperate file and only loaded on the server as it is bloating the clientside bundle a lot. Give me a shout if you need some help as I have a few ideas

Reply:
Decided to remove the polyfill which reduced the bundle size.